src/codegen.cpp

```diff
@@ -4422,6 +4422,9 @@ static std::unique_ptr<Module> emit_function(
         // TODO: Fix when moving to new LLVM version
         #if JL_LLVM_VERSION < 30400
         dbuilder.createCompileUnit(0x01, filename, ".", "julia", true, "", 0);
+        #elif JL_LLVM_VERSION >= 40000
```

Comment:
I do like no indent better and I'm just making it consistent with whatever around it.... Should I change this?